www/images/real-experience-icon.svg

```diff
@@ -1 +1 @@
-<svg width="110" height="110" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16 27h78a1 1 0 011 1v49H15V28a1 1 0 011-1zm-4 50V28a4 4 0 014-4h78a4 4 0 014 4v49h7a1 1 0 011 1v7a4 4 0 01-4 4H8a4 4 0 01-4-4v-7a1 1 0 011-1h7zm86 3H7v5a1 1 0 001 1h94a1 1 0 001-1v-5h-5zM47 50c0-5.632 4.531-9 9-9s9 3.368 9 9c0 7.141-4.569 10.762-7.834 13.35l-.541.43-.68.544-.632-.597a45.872 45.872 0 00-1.352-1.207C51.028 59.967 47 56.462 47 50zm9-7c-3.531 0-7 2.632-7 7 0 5.481 3.252 8.341 6.174 10.911.295.26.587.517.873.773C59.3 59.094 63 56.035 63 50c0-4.368-3.469-7-7-7zm2 7a2 2 0 11-4 0 2 2 0 014 0zm2 0a4 4 0 11-8 0 4 4 0 018 0z" fill="#fff"/><path fill-rule="evenodd" clip-rule="evenodd" d="M16 27h78a1 1 0 011 1v49H15V28a1 1 0 011-1zm-4 50V28a4 4 0 014-4h78a4 4 0 014 4v49h7a1 1 0 011 1v7a4 4 0 01-4 4H8a4 4 0 01-4-4v-7a1 1 0 011-1h7zm86 3H7v5a1 1 0 001 1h94a1 1 0 001-1v-5h-5zM47 50c0-5.632 4.531-9 9-9s9 3.368 9 9c0 7.141-4.569 10.762-7.834 13.35l-.541.43-.68.544-.632-.597a45.872 45.872 0 00-1.352-1.207C51.028 59.967 47 56.462 47 50zm9-7c-3.531 0-7 2.632-7 7 0 5.481 3.252 8.341 6.174 10.911.295.26.587.517.873.773C59.3 59.094 63 56.035 63 50c0-4.368-3.469-7-7-7zm2 7a2 2 0 11-4 0 2 2 0 014 0zm2 0a4 4 0 11-8 0 4 4 0 018 0z" fill="url(#paint0_linear)"/><rect x="77.5" y="49.5" width="27" height="47" rx="2.5" fill="#121D35" stroke="#A8C8FD" stroke-width="3"/><circle cx="91" cy="89" r="2" fill="#A8C8FD"/><defs><linearGradient id="paint0_linear" x1="55" y1="29" x2="55" y2="82" gradientUnits="userSpaceOnUse"><stop stop-color="#4381E5"/><stop offset="1" stop-color="#AFCEFF"/></linearGradient></defs></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="110" height="110" fill="none" stroke="url(#a)" stroke-width="3"><linearGradient id="a" x2="0" y1="29" y2="82" gradientUnits="userSpaceOnUse"><stop stop-color="#4381e5"/><stop offset="1" stop-color="#afceff"/></linearGradient><path stroke-linejoin="round" d="M13.5 77V28a2.5 2.5 0 0 1 2.5-2.5h78a2.5 2.5 0 0 1 2.5 2.5v21M77 78.5H5.5V85A2.5 2.5 0 0 0 8 87.5h69M91 89v.1"/><path stroke-width="2" d="M48 50c0 5.6 2.9 8.8 8 13 4.5-3.4 8-7 8-13a8 8 0 1 0-16 0zm5 0c0-4 6-4 6 0s-6 4-6 0z"/><rect width="27" height="47" x="77.5" y="49.5" rx="2.5"/><circle cx="91" cy="89" r=".5"/></svg>
```

Comment:
You’re right, it is. I assumed the color was chosen to match the backdrop of the icon’s use on the homepage, because if you remove the `fill` on the `<rect>` that draws the phone shape, the lines of the computer behind it show through, so I removed that geometry behind it along with the `fill`.

However, if I squint at the usage on the homepage…

<img width="175" alt="The icon over a navy blue backdrop." src="https://user-images.githubusercontent.com/8072522/157347942-35525a9b-2d43-4eaa-b34f-aa4d8fa4b68b.png">

The phone screen is very subtly different: `#121d35` vs. the `background-color`’s `#182437`. That could be intentional, or it could be an artifact of color-space handling between the original art program vs. browsers.

If that difference is unintentional, the current suggested replacement has the bonus of letting this icon be easily reused on any background. I guess that’s a question for design?